lib/chef-acceptance/chef_runner.rb

```diff
@@ -74,19 +74,17 @@ def chef_config_template
           #{File.expand_path('..', acceptance_cookbook.root_dir).inspect},
           #{File.expand_path('../../../.shared', acceptance_cookbook.root_dir).inspect}
         ]
-        node_path #{File.expand_path('nodes', acceptance_cookbook.root_dir).inspect}
+        node_path "#{node_path}"
         stream_execute_output true
         audit_mode #{app_options.audit_mode ? ":enabled" : ":disabled"}
```

Comment:
Should we add some more of the Chef config values here? At the very least we probably want the following represented:

* `file_cache_path`
* `checksum_path`

I would refer back to what TK does for other examples:
https://github.com/test-kitchen/test-kitchen/blob/master/lib/kitchen/provisioner/chef_base.rb#L193-L218